sdk/formrecognizer/Azure.AI.FormRecognizer/src/ServiceClient.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Custom;
+using Azure.AI.FormRecognizer.Models;
+using Azure.Core;
+
+namespace Azure.AI.FormRecognizer
+{
+    [CodeGenClient("")]
+    internal partial class ServiceClient
+    {
+        #region Custom
+
+        internal ResponseWithHeaders<AnalyzeWithCustomModelHeaders> AnalyzeWithCustomModel(Guid modelId, bool? includeTextDetails, Stream stream, FormContentType contentType, CancellationToken cancellationToken = default)
```

Comment:
It would be nice to file issues for reasons why you had to workaround and reference them here so we can remove workarounds when issues are fixed.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/10390